[BUG] [Beta] Error in password removal workflow action

### Description

Thanks a ton to everyone involved in this huge release. It looks awesome!
I tried out the new password removal workflow action, but whether I supply one or multiple passwords (one per line), consumption fails with the following error:

`expected string or bytes-like object, got 'list'` (see full error below)

### Steps to reproduce

1. Setup Workflow with Password removal action
2. Enter one or multiple passwords, separated by line break
3. Throw a file in the consumer

<img width="829" height="394" alt="Image" src="https://github.com/user-attachments/assets/f9f2a728-54e0-43ea-9c5e-1d8cf11c6a43" />

### Webserver logs

```bash
[2026-05-05 20:12:19,885] [ERROR] [paperless.tasks] [fa06d6e8] WorkflowTriggerPlugin failed: expected string or bytes-like object, got 'list'
Traceback (most recent call last):
  File "/usr/src/paperless/src/documents/tasks.py", line 185, in consume_file
    msg = plugin.run()
          ^^^^^^^^^^^^
  File "/usr/src/paperless/src/documents/consumer.py", line 79, in run
    overrides, msg = run_workflows(
                     ^^^^^^^^^^^^^^
  File "/usr/src/paperless/src/documents/signals/handlers.py", line 959, in run_workflows
    execute_password_removal_action(action, document, logging_group)
  File "/usr/src/paperless/src/documents/workflows/actions.py", line 295, in execute_password_removal_action
    for password in re.split(r"[,\n]", passwords)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/re/__init__.py", line 207, in split
    return _compile(pattern, flags).split(string, maxsplit)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: expected string or bytes-like object, got 'list'
[2026-05-05 20:12:19,902] [ERROR] [celery.app.trace] Task documents.tasks.consume_file[fa06d6e8-a25a-46d7-9bc2-1e93f52e1c33] raised unexpected: TypeError("expected string or bytes-like object, got 'list'")
Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/celery/app/trace.py", line 479, in trace_task
    R = retval = fun(*args, **kwargs)
                 ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/celery/app/trace.py", line 779, in __protected_call__
    return self.run(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/asgiref/sync.py", line 361, in main_wrap
    raise exc_info[1]
  File "/usr/src/paperless/src/documents/tasks.py", line 185, in consume_file
    msg = plugin.run()
          ^^^^^^^^^^^^
  File "/usr/src/paperless/src/documents/consumer.py", line 79, in run
    overrides, msg = run_workflows(
                     ^^^^^^^^^^^^^^
  File "/usr/src/paperless/src/documents/signals/handlers.py", line 959, in run_workflows
    execute_password_removal_action(action, document, logging_group)
  File "/usr/src/paperless/src/documents/workflows/actions.py", line 295, in execute_password_removal_action
    for password in re.split(r"[,\n]", passwords)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/re/__init__.py", line 207, in split
    return _compile(pattern, flags).split(string, maxsplit)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: expected string or bytes-like object, got 'list'
```

### Browser logs

```bash

```

### Services logs

```bash

```

### Paperless-ngx version

3.0.0-beta

### Host OS

Synology DSM 7 | x86_64

### Installation method

Docker - official image

### System status

```json
{
    "pngx_version": "2.20.15",
    "server_os": "Linux-4.4.302+-x86_64-with-glibc2.41",
    "install_type": "docker",
    "storage": {
        "total": 1546188226560,
        "available": 803334250496
    },
    "database": {
        "type": "postgresql",
        "url": "paperless",
        "status": "OK",
        "error": null,
        "migration_status": {
            "latest_migration": "paperless_mail.0001_squashed",
            "unapplied_migrations": []
        }
    },
    "tasks": {
        "redis_url": "redis://paper-beta-redis:6379",
        "redis_status": "OK",
        "redis_error": null,
        "celery_status": "OK",
        "celery_url": "celery@paperless-ngx-beta",
        "celery_error": null,
        "index_status": "OK",
        "index_last_modified": "2026-05-05T20:00:41.750758+02:00",
        "index_error": null,
        "classifier_status": "OK",
        "classifier_last_trained": "2026-05-05T18:19:17.023441Z",
        "classifier_error": null,
        "sanity_check_status": "WARNING",
        "sanity_check_last_run": null,
        "sanity_check_error": "No sanity check tasks found",
        "llmindex_status": "DISABLED",
        "llmindex_last_modified": null,
        "llmindex_error": null,
        "summary": {
            "days": 30,
            "total_count": 8,
            "pending_count": 0,
            "success_count": 5,
            "failure_count": 2
        }
    },
    "websocket_connected": "OK"
}
```

### Browser

Win Edge

### Configuration changes

_No response_

### Please confirm the following

- [x] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [x] This issue is not about the OCR or archive creation of a specific file(s). Otherwise, please see above regarding OCR tools.
- [x] I have already searched for relevant existing issues and discussions before opening this report.
- [x] I have updated the title field above with a concise description.

--- a/src/documents/tests/test_workflows.py
+++ b/src/documents/tests/test_workflows.py
@@ -4164,7 +4164,7 @@ class TestWorkflows(
         )
         action = WorkflowAction.objects.create(
             type=WorkflowAction.WorkflowActionType.PASSWORD_REMOVAL,
-            passwords="wrong, right\n extra ",
+            passwords=["wrong", "right", "extra"],
         )
         workflow = Workflow.objects.create(name="Password workflow")
         workflow.triggers.add(trigger)
@@ -4218,7 +4218,7 @@ class TestWorkflows(
         )
         action = WorkflowAction.objects.create(
             type=WorkflowAction.WorkflowActionType.PASSWORD_REMOVAL,
-            passwords=" \n , ",
+            passwords=[" ", "  "],
         )
         workflow = Workflow.objects.create(name="Password workflow missing passwords")
         workflow.triggers.add(trigger)
@@ -4276,7 +4276,7 @@ class TestWorkflows(
         """
         action = WorkflowAction.objects.create(
             type=WorkflowAction.WorkflowActionType.PASSWORD_REMOVAL,
-            passwords="first, second",
+            passwords=["first", "second"],
         )
 
         temp_dir = Path(tempfile.mkdtemp())

--- a/src/documents/workflows/actions.py
+++ b/src/documents/workflows/actions.py
@@ -1,5 +1,4 @@
 import logging
-import re
 import uuid
 from pathlib import Path
 
@@ -290,11 +289,7 @@ def execute_password_removal_action(
         )
         return
 
-    passwords = [
-        password.strip()
-        for password in re.split(r"[,\n]", passwords)
-        if password.strip()
-    ]
+    passwords = [p.strip() for p in passwords if p.strip()]
 
     if isinstance(document, ConsumableDocument):
         # hook the consumption-finished signal to attempt password removal later

